packages/account-management/src/frontend/Registration/Registration.tsx

```diff
@@ -64,7 +53,7 @@ export const Registration: React.FC = () => {
   }, [pass]);
 
   useEffect(() => {
-    setValid(firstName && lastName && email && passwordPolicy.test(pass || ''));
+    setValid(!!(firstName && lastName && email && passwordPolicy.test(pass || '')));
```

Comment:
What's this `!!` operator? It's in a few places in here but I've had a google but the only thing I can understand it as being is `not not` which would be redundant...

Reply:
`not not` is right, but it's a quick and dirty way to convert something to a boolean, equivalent to `Boolean(firstName && lastName && ...`